src/pybind/mgr/progress/module.py

```diff
@@ -652,6 +652,7 @@ def clear(self):
         self._completed_events = []
         self._dirty = True
         self._save()
+        self.clear_all_progress_events()
```

Comment:
@kamoltat aren't we effectively doing 6f60d33115d2f583331d31b95a0a33b96a614f09? If so, it is better to just cherry-pick the commit.

Reply:
I agree